_includes/page-footer.html

```diff
@@ -1,9 +1,11 @@
 <footer class="site-footer">
-  <span class="site-footer-owner"><a href="{{ site.url }}">{{ site.title }}</a> is maintained by <a href="{{ site.author.url }}">{{ site.author.name }}</a>.</span>
-  <span class="site-footer-credits">This page was generated by <a
-href="https://pages.github.com">GitHub Pages</a>.
+  <span class="site-footer-owner">
+    <a href="{{ site.url }}">{{ site.title }}</a> von <a href="{{ site.author.url }}">{{ site.author.name }}</a>.<br />
+    Sketchnotes von <a href="https://twitter.com/Teapot4181">Lisa Maria Moritz</a>
+  </span>
+  <nav>
     <a href="/datenschutzhinweise.html">Datenschutzhinweise</a>
     <a href="/impressum.html">Impressum</a>
-  </span>
-
+  </nav>
+  <span class="site-footer-credits" lang="en">This page was generated by <a href="https://pages.github.com">GitHub Pages</a>.</span>
 </footer>
```

Comment:
Aisde: I changed file permissions here (`chmod -x`), but opted not to remove all the other files' executable flag where it seems unwarranted.

Reply:
I fixed the other file permissions in 7cc4072db7e4b52192dca4bf6217b2371d3fddb1 . This might be related to problems with accessing WLS filesystems from Windows. Thanks for pointing this out!